### ISSUE TITLE
Not compatible with Cinnamon 3.4

### ISSUE THREAD

**Issue by yochananmarqos** [germanfr/cinnamon-transparent-panels#10]
From glass.log:

```
error t=2017-07-04T01:32:16.009Z [Extension "transparent-panels@germanfr"]: Extension is not compatible with current Cinnamon version
error t=2017-07-04T01:32:16.010Z Could not load extension transparent-panels@germanfr
```

**Comment by germanfr** [comment]
Thanks for reporting :+1:

**Comment by venatiodecorus** [comment]
I'm having this issue. This was just closed with no resolution? Is there a fix for this?

**Comment by germanfr** [comment]
This was fixed here 084bd1f as the close tag indicates. Update the extension to the latest version.

**Comment by germanfr** [comment]
If you are still on Cinnamon 3.4 the latest version will probably not work. In that case download it from [here](https://github.com/germanfr/cinnamon-transparent-panels/releases/tag/3.4) for your version.
